support-frontend/package.json

```
@@ -27,7 +27,7 @@
     "jest-update-snapshot": "echo 'Updating JS tests' && jest -u",
     "orphans": "echo 'Checking for orphan modules' && node scripts/orphans.js",
     "check-images": "echo 'Purging unused images' && node scripts/gridImages.js",
-    "ts-errors": "node scripts/ts-log-errors.js"
+    "ts-errors": "node scripts/ts-log-errors.js && ./scripts/generate-history-csv.sh > ./ts-error-history.csv"
```

I think there can be some issue with execute permission hence we used bash directly, although I forgot what now
```suggestion
    "ts-errors": "node scripts/ts-log-errors.js && bash scripts/generate-history-csv.sh > ./ts-error-history.csv"
```

---

👍